docs/package_saplings.md

```diff
@@ -0,0 +1,33 @@
+Packaging Saplings
+------------------
+
+To make distribution easlier, saplings are packaged as bzipped tar files with
+the extension ".sapling". A Docker Compose file that creates packages for the
+saplings in this repo can be found in the `ci/` directory. Because the packaging
+process takes place in Docker containers, both Docker and Docker Compose are
+required to run the following commands.
+
+To generate sapling packages, from the root of the `splinter-ui` repository run:
```

Comment:
Need a comma after "repository"

Or you could reword the sentence: "To generate sapling packages, run this command from the root of the ...")